lib/auto-languageclient.js

```diff
@@ -572,4 +561,12 @@ export default class AutoLanguageClient {
   filterChangeWatchedFiles(filePath: string): boolean {
     return true;
   }
+
+  /**
+   * Called on language server stderr output.
+   * @param stderr a chunk of stderr from a language server instance
+   */
+  handleServerStderr(stderr: string, projectPath: string) {
+    this.logger.debug(`stderr ${stderr}`);
+  }
```

Comment:
I'm okay with us logging by default.  I wonder if we should chose .error ?

Reply:
It was _warn_ in the unused code previously. I don't mind!

Reply:
This should be for each line too, the console handles it much better.